node/core/dispute-coordinator/src/scraping/mod.rs

```diff
@@ -182,28 +190,49 @@ impl ChainScraper {
 	where
 		Sender: overseer::DisputeCoordinatorSenderTrait,
 	{
-		// Get included events:
-		let included =
+		// Get included and backed events:
+		let events =
 			get_candidate_events(sender, block_hash)
 				.await?
 				.into_iter()
-				.filter_map(|ev| match ev {
-					CandidateEvent::CandidateIncluded(receipt, _, _, _) => Some(receipt),
-					_ => None,
+				.filter(|ev| match ev {
+					CandidateEvent::CandidateIncluded(_, _, _, _) => true,
+					CandidateEvent::CandidateBacked(_, _, _, _) => true,
+					_ => false,
 				});
-		for receipt in included {
-			let candidate_hash = receipt.hash();
-			gum::trace!(
-				target: LOG_TARGET,
-				?candidate_hash,
-				?block_number,
-				"Processing included event"
-			);
-			self.included_candidates.insert(candidate_hash);
-			self.candidates_by_block_number
-				.entry(block_number)
-				.or_default()
-				.insert(candidate_hash);
+		for ev in events {
+			match ev {
+				CandidateEvent::CandidateIncluded(receipt, _, _, _) => {
+					let candidate_hash = receipt.hash();
+					gum::trace!(
+						target: LOG_TARGET,
+						?candidate_hash,
+						?block_number,
+						"Processing included event"
+					);
+					self.included_candidates.insert(candidate_hash);
+					self.candidates_by_block_number
+						.entry(block_number)
+						.or_default()
+						.insert(candidate_hash);
+				},
+				CandidateEvent::CandidateBacked(receipt, _, _, _) => {
+					let candidate_hash = receipt.hash();
+					gum::trace!(
+						target: LOG_TARGET,
+						?candidate_hash,
+						?block_number,
+						"Processing backed event"
+					);
+					self.backed_candidates.insert(candidate_hash);
+				},
+				_ => {
```

Comment:
Yeah, but why filter, if we have this loop anyways? I would just get rid of the filter at the top, it adds no value.

Reply:
True. It's pointless.